### ISSUE TITLE
ViewScene Record Movie UI upgrades

### ISSUE THREAD

**Issue by a-y-khan** [SCIInstitute/SCIRun#279]
Replace file name text entry with file selection dialog. Make sure outstanding bugs are fixed (see GForge). The radio button controls shouldn't include the stop recording function (move to button?).

**Comment by dcwhite** [comment]
@jessdtate Good one for interns? Now that the screenshot utility is getting worked on...movies might not be far off

**Comment by jessdtate** [comment]
yeah.  I think the same hangups apply to auto screenshots.  if we can sort that out, video stacks should be pretty easy.

**Comment by dcwhite** [comment]
@RubioJr9 @Haydelj Coming soon, SCIRun movies!

**Comment by dcwhite** [comment]
**Triage 2026-06:** ViewScene Record Movie UI — the text entry for output path should be replaced with a `QFileDialog`. The relevant code is in the ViewScene config dialog. The screenshot path is already handled via `QFileDialog` in the screenshot code path; the same pattern should be applied to the movie path widget. Low effort.
